scripts/gulp/test.js

```diff
@@ -66,7 +68,7 @@ gulp.task( 'test:unit', unitTest );
 gulp.task( 'test:build', buildTest );
```

Comment:
Trivial: Mind updating the names to `testUnit`, `testBuild`,  and `testQUnit` while you're in here? (was my mistake).

Reply:
Yeah.